bindings/python/jsonschema_rs.pyi

```diff
@@ -0,0 +1,69 @@
+from typing import Any, TypeVar
+from collections.abc import Iterator
```

Comment:
Should it be from `typing` as well?

Reply:
It's already deprecated https://docs.python.org/3/library/typing.html#typing.Iterator

Reply:
Thanks!